python_modules/libraries/dagster-dbt/dagster_dbt/cloud_v2/client.py

```diff
@@ -22,8 +22,8 @@
     os.getenv("DAGSTER_DBT_CLOUD_BATCH_RUNS_REQUEST_LIMIT", "100")
 )
 DAGSTER_ADHOC_TRIGGER_CAUSE = "Triggered by dagster."
-DEFAULT_POLL_INTERVAL = 1
-DEFAULT_POLL_TIMEOUT = 60
+DAGSTER_DBT_CLOUD_POLL_INTERVAL = int(os.getenv("DAGSTER_DBT_CLOUD_POLL_INTERVAL", "1"))
+DAGSTER_DBT_CLOUD_POLL_TIMEOUT = int(os.getenv("DAGSTER_DBT_CLOUD_POLL_TIMEOUT", "60"))
```

Comment:
should these not be options on `DbtCloudWorkspaceClient` ?

Reply:
Yeah I can change that in a subsequent PR